.circleci/config.yml

```
@@ -15,34 +15,30 @@ steps-test: &steps-test
               curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
               export NVM_DIR="$HOME/.nvm"
               [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-              nvm install 10
-              nvm alias default 10
+              nvm install 12
+              nvm alias default 12
               echo 'export NVM_DIR=${HOME}/.nvm' >> $BASH_ENV
               echo "[ -s '${NVM_DIR}/nvm.sh' ] && . '${NVM_DIR}/nvm.sh'" >> $BASH_ENV
               ;;
             Windows*|CYGWIN*|MINGW*|MSYS*)
-              nvm install 10.22.0
-              nvm use 10.22.0
+              nvm install 12.22.1
+              nvm use 12.22.1
```

The Windows CI worker is giving some extremely strange errors, going to try a different version here
```suggestion
              nvm install 12.22.4
              nvm use 12.22.4
```